openedx/core/lib/edx_api_utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 def get_edx_api_data(api_config, user, resource,
-                     api=None, resource_id=None, querystring=None, cache_key=None):
+                     api=None, resource_id=None, querystring=None, cache_key=None, collection=True):
```

Comment:
the term "collection" threw me here -- I'd prefer `multiple=True` or `many=True` unless there's a need to go with `collection=True`